dds/DCPS/SporadicTask.h

```diff
@@ -29,24 +29,42 @@ class SporadicTask : public RcEventHandler {
 
   void schedule(const TimeDuration& delay)
   {
-    interceptor_->execute_or_enqueue(new ScheduleCommand(this, delay));
+    RcHandle<ReactorInterceptor> interceptor = interceptor_.lock();
+    if (interceptor) {
+      interceptor->execute_or_enqueue(new ScheduleCommand(this, delay));
+    } else {
+      ACE_ERROR((LM_ERROR, "(%P|%t) SporadicTask::schedule"
```

Comment:
Downgrade these to warnings.  Eliminate the %p since errno will most likely not be a factor.